extras/index.js

```diff
@@ -16,3 +16,4 @@ export { RenderPassCompose } from './render-passes/render-pass-compose.js';
 export { RenderPassDownSample } from './render-passes/render-pass-downsample.js';
 export { RenderPassUpSample } from './render-passes/render-pass-upsample.js';
```

Comment:
Unrelated to this PR but I just noticed that `UpSample` should really be `Upsample`. See: https://en.wikipedia.org/wiki/Upsampling

Reply:
Noted, thanks, will rename those next year.